src/App/Components/Comment/comment.icons.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+import React from 'react';
+
+const CommentIcons = (props) => {
+  const { edit } = props;
+  return (
+    <div className="comment-icons">
+      <div className="comment-icons__dislike-comment" title="dislike">
+        <i className="fas fa-heart-broken" id="dislike" />
+        {' '}
+        <small id="number">69</small>
+      </div>
+
+      <div className="comment-icons__dislike-comment ml-2" title="like">
+        <i className="fas fa-heart" id="like" />
+        {' '}
+        <small id="number">69</small>
```

Comment:
@MCFrank16 how come these numbers are static?

Reply:
I haven't yet worked on like and dislike, I will change them in my next task that I should start right after this